jwst/master_background/master_background_mos_step.py

```diff
@@ -107,7 +107,7 @@ def process(self, data):
             if not self.force_subtract and \
                'COMPLETE' in [data_model.meta.cal_step.back_sub, data_model.meta.cal_step.master_background]:
                 self.log.info('Background subtraction has already occurred. Skipping.')
-                self.record_step_status(data, 'master_background', False)
+                self.record_step_status(data, 'master_background_mos', False)
```

Comment:
We only have 1 keyword "S_MSBSUB" to hold the step completion status for both `master_background` and `master_background_mos`. Does that work OK?  Will the `record_step_status` function be able to accept 2 different step names as input to update the value of the same keyword?

Reply:
Good catch - I hamfisted the find-and-replace and didn't check results because of regtest/CRDS woes... Running spec2 on some regtest MOS data shows that this leaves S_MSBSUB off the list of calibration step keywords listed in the header.